core/bin/external_node/src/main.rs

```diff
@@ -94,8 +94,8 @@ async fn build_state_keeper(
         stop_receiver_clone.changed().await?;
         result
     }));
-    let batch_executor_base: Box<dyn BatchExecutor> =
-        Box::new(MainBatchExecutor::new(save_call_traces, true));
```

Comment:
This actually changes nothing. Committed by accident?

Reply:
Ah, the variable name also changed.

Reply:
Yes, exactly, it's a cosmetic change shortening the var name.